### ISSUE TITLE
More problems with Claymore Dual Miner

### ISSUE THREAD

**Issue by Pandoraaaaa** [MultiPoolMiner/MultiPoolMiner#1704]
Really sorry to be a pest, but I am determined to get this miner running. If you remember, the binary download does not work properly so I manually installed the contents of this zip file:
```
Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak.AMD.NVIDIA.GPU.Miner.v11.7.-.Catalyst.15.12-18.x.-.CUDA.8.0_9.1_7.5_6.5.zip
```
into .\Bin\Ethash-Claymore

I then ran the benchmarks and everything is smooth, which is amazing, except that one stinker. And Eth/Pascal dual mining was my big winner with the official release version of mpm. I need to get this miner running or revert out of the master code base. Anyway here is the error message that I am getting every time I start mpm:
```
Unblock-File : C:\Users\miria\Desktop\mpm\Bin\Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak.AMD.NVIDIA.GPU.Miner.v11.7.-.Catalyst.15.12
-18.x.-.CUDA.8.0_9.1_7.5_6.5\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak AMD+NVIDIA GPU Miner v11.7
At C:\Users\miria\Desktop\mpm\MultiPoolMiner.ps1:104 char:91
+ ... rorAction SilentlyContinue) {Get-ChildItem . -Recurse | Unblock-File}
+                                                             ~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\miria\...GPU Miner v11.7:String) [Unblock-File], FileNotFoundException
    + FullyQualifiedErrorId : FileNotFound,Microsoft.PowerShell.Commands.UnblockFileCommand

Get-ChildItem : Could not find a part of the path 'C:\Users\miria\Desktop\mpm\Bin\Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak.AMD.NVI
DIA.GPU.Miner.v11.7.-.Catalyst.15.12-18.x.-.CUDA.8.0_9.1_7.5_6.5\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak AMD+NVIDIA GPU Miner
v11.7'.
At C:\Users\miria\Desktop\mpm\MultiPoolMiner.ps1:104 char:64
+ ... ile" -ErrorAction SilentlyContinue) {Get-ChildItem . -Recurse | Unblo ...
+                                          ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ReadError: (C:\Users\miria\...GPU Miner v11.7:String) [Get-ChildItem], DirectoryNotFoundException
    + FullyQualifiedErrorId : DirIOError,Microsoft.PowerShell.Commands.GetChildItemCommand
```
So it seems the software is looking for a file or folder with that crazy long name underneath .\bin (rather than looking for .\bin\Ethash-Claymore\EthDcrMiner64.exe

Here is the line in at MultiPoolMiner.ps1:104 char:64 where the error occurs:
```
if (Get-Command "Unblock-File" -ErrorAction SilentlyContinue) {Get-ChildItem . -Recurse | Unblock-File}
```
Thank you very much for looking at this, and other than this one issue everything is running great for me.

**Comment by UselessGuru** [comment]
> So it seems the software is looking for a file or folder with that crazy long name underneath .\bin (rather than looking for .\bin\Ethash-Claymore\EthDcrMiner64.exe

This is probably caused by the overly long file name (which we sill change soon).

Please try this:
1. Shorten the long file name to somethin like 'ClaymoreDual_11_7.zip'
2. If \bin\Ethash-Claymore\EthDcrMiner64.exe is not present manually unpack the Zip file
3. Restart MPM. The problem should be gone. 

Please report your findings here. TIA!

**Comment by Pandoraaaaa** [comment]
Sorry, that change had no effect and the error message was the same, referring to that prev long name. Where is the list of binary repositories/paths stored? Don't we need to edit that as well?

**Comment by UselessGuru** [comment]
What files are in bin\Ethash-Claymore\?

If EthDcrMiner64.exe is there then delete the long file - it is no longer needed.
MPM checks for the exe file in the bin folder, verifies the hash. When this matches then the ZIP file is no longer needed.

**Comment by Pandoraaaaa** [comment]
Ok, here is the scoop:
1. I changed all occurances of the long filename in the .\miners folder
2. I renamed and then removed a folder under .\bin that had the long filename. Windows 10 was unable to delete it until I named it "blah.zip" due to the name being too long - that folder was causing the error in my initial post in this thread
3. I deleted all files in stats with word Claymore in them
4. Restart, 57 miners to benchmark
So we'll see how it goes. Hopefully some claymore dual mining starts up.

Let me know if you want me to commit and push the change up here. I don't know if I have permission to do that or to make a pull request. Anyway I made the name "ClaymoreDual_11_7.zip" as you suggested. I guess you could merger that into master after the name gets changed here:
```
 https://github.com/MultiPoolMiner/miner-binaries/releases/download/ethdcrminer64/Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak.AMD.NVIDIA.GPU.Miner.v11.7.-.Catalyst.15.12-18.x.-.CUDA.8.0_9.1_7.5_6.5.zip
```

**Comment by Pandoraaaaa** [comment]
The Ethhash-Claymore folder contains the unzipped contents of
```
 https://github.com/MultiPoolMiner/miner-binaries/releases/download/ethdcrminer64/Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal_Blake2s_Keccak.AMD.NVIDIA.GPU.Miner.v11.7.-.Catalyst.15.12-18.x.-.CUDA.8.0_9.1_7.5_6.5.zip
```

**Comment by mkirov0** [comment]
I just downloaded a fresh copy and thats what I have:
<img width="247" alt="screen shot 2018-05-13 at 1 05 43 pm" src="https://user-images.githubusercontent.com/28104701/39966078-6f0269a4-56ae-11e8-973a-4fab0c6f08e1.png">

**Comment by UselessGuru** [comment]
@Pandoraaaaa 

Pls stop changing files unless you are exactly sure what you are doing. 
There is absolutely NO reason to change any files at all.

Please retore everything back to original.
If ClaymoreDual is causing problems than download the miner manually, rename the ZIP file and unpack the contents to bin\Ethash-Claymore. Then delete the ZIP file. Done.

**Comment by Pandoraaaaa** [comment]
Haha, I'm not changing your repo, sorry to freak you out there. The overlong filename was causing issues here for me on Windows 10 and I fixed it. 

I've got everything working here on my computers. Thank you for your help.
